nupic/algorithms/anomaly.py

```diff
@@ -72,10 +72,11 @@ class Anomaly(object):
   _supportedModes = (MODE_PURE, MODE_LIKELIHOOD, MODE_WEIGHTED)
 
 
-  def __init__(self, slidingWindowSize=None, mode=MODE_PURE, binaryAnomalyThreshold=None):
+  def __init__(self, slidingWindowSize=0, mode=MODE_PURE, binaryAnomalyThreshold=0.0):
```

Comment:
But then you're forced to swarm over parameters from `0..N` rather than the range you want, which may be `20..50`. I'd rather change the permute variable to have a `tryNone` option which determines whether it tries `None` in addition to some integer/float range.